lib/job-iteration/integrations/sidekiq.rb

```diff
@@ -4,11 +4,19 @@
 
 module JobIteration
   module Integrations # @private
-    JobIteration.interruption_adapter = -> do
-      if defined?(Sidekiq::CLI) && Sidekiq::CLI.instance
-        Sidekiq::CLI.instance.launcher.stopping?
-      else
-        false
+    module Sidekiq
+      extend self
+
+      attr_accessor :stopping
+
+      alias_method :call, :stopping
```

Comment:
I'd prefer to be a little less fancy here, and only define the things we need on the singleton class, not instances as well.

```suggestion
      class << self
        attr_accessor :stopping

        def call
          stopping
        end
      end
```

Reply:
Fixed!